src/target/efm32.c

```diff
@@ -1000,19 +967,19 @@ void efm32_aap_probe(ADIv5_AP_t *ap)
 
 	/* New target */
 	target *t = target_new();
+	if (!t) {
+		return;
+	}
+
 	adiv5_ap_ref(ap);
```

Comment:
It looks like we might have forgotten to assign t->mass_erase here - please check and verify the mass erase behaviour of the AAP

Reply:
you're thinking ahead, this is not yet rebased on the mass erase unification

Reply:
Ah, that'd do it! Resolving for now with a note to check again when it has been.